toko/templates/products/detail.html

```diff
@@ -1,43 +1,50 @@
 {% extends 'base.html' %}
 {% block content %}
+<div class="container-fluid">
+<div class="content-wrapper">
+  <div class="item-container">
+    <div class="container">
+      <div class="col-md-8">
+        <h2>{{ product }} </h2>
+        {% for photo in product.photos.all %}
```

Comment:
class ProductVariation(models.Model):
         product = models.ForeignKey('products.Product',
related_name='variations')

    class Photo(TimeStampedModel):
         product = models.ForeignKey('products.Product',
related_name='photos')


bukanya ini one to many, sory masih nubie

On Mon, Feb 22, 2016 at 2:42 PM, Sutrisno Efendi <notifications@github.com>
wrote:

> In toko/templates/products/detail.html
> <https://github.com/ular/toko/pull/7#discussion_r53593691>:
>
> > @@ -1,43 +1,50 @@
> >  {% extends 'base.html' %}
> >  {% block content %}
> > +<div class="container-fluid">
> > +<div class="content-wrapper">
> > +  <div class="item-container">
> > +    <div class="container">
> > +      <div class="col-md-8">
> > +        <h2>{{ product }} </h2>
> > +        {% for photo in product.photos.all %}
>
> Kalo fieldnya many-to-many atau many-to-one kita pake prefetch_related,
> sedangkan kalo one-to-many kita pake select_related
>
>
> https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.prefetch_related
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ular/toko/pull/7/files#r53593691>.
>